.github/workflows/windows_workflow.yml

```diff
@@ -0,0 +1,42 @@
+name: ubuntu-workflow
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - '*'
+    paths:
+      - '**'
+      - '!flank-scripts/**'
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          submodules: true
+      - name: Display the java version
+        run: java -version 
+        shell: bash
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Gradle check
+        uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-executable: "./test_runner/gradlew"
+          arguments: "-p test_runner check"
+
+      - name: Gradle clean build
+        uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-executable: "./test_runner/gradlew"
+          arguments: "-p test_runner clean build"
```

Comment:
gradle `build` run gradle `check` under the hood

`build` = `assemble` + `check`

Reply:
ill remove the check then 👍